build-cran-repo/README.md

```diff
@@ -0,0 +1,56 @@
+# build-cran-repo
+
+This action builds a list of R packages for WebAssembly and creates a CRAN-like repository containing the R package binaries.
+
+This action uses the [rwasm](https://r-wasm.github.io/rwasm/) R package to build packages for WebAssembly, which must be run in Docker container `ghcr.io/r-wasm/webr:main` to ensure access to a Wasm development toolchain. By requiring the step to use Docker, the action can only be run in `ubuntu` based runners.
+
+## Inputs
+
+* **packages** (*required*) - A string of [R package references](https://r-lib.github.io/pkgdepends/reference/pkg_refs.html).
+* **repo-path** (`_site`) - The path to the directory where the R package library filesystem image files should be saved.
+  * Note: Due to limitations in GitHub Actions, the saved files can not be altered in any subsquent steps. Related: [actions/runner#434](https://github.com/actions/runner/issues/434).
+
+## Steps
+
+1. Install latest `r-wasm/rwasm`
+2. Create the repo using `{rwasm}`
+3. Copy files to the `repo-path` directory
+
+## Usage
+
+```yaml
+jobs:
+  repo:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build wasm repo
+      uses: r-wasm/actions/build-cran-repo@v1
+      with:
+        packages: |
+          cli
+        repo-path: out-repo
+```
+
+Typically, the repository is then used to create a GitHub Pages site. This can be done using the [`deploy-cran-repo`](../examples/deploy-cran-repo.yml) workflow example.
```

Comment:
```suggestion
Typically, the resulting CRAN-like repository is then used to create a GitHub Pages site. This can be done using the [`deploy-cran-repo`](../examples/deploy-cran-repo.yml) workflow example.

The package repository can alternatively be deployed to other static storage hosting in subsequent steps.
```

Should we mention something along these lines for users deploying to other hosting providers (e.g. S3, CloudFlare, Netlify, ...)?

I guess those users would upload the contents of `out-repo` using some other custom step after this one.